hikari/impl/cache.py

```diff
@@ -72,11 +95,11 @@ class StatefulCacheImpl(cache.MutableCache):
         "_unknown_custom_emoji_entries",
         "_user_entries",
         "_message_entries",
+        "_referenced_messages",
```

Comment:
you need to add the missing slots to MutableCache

Reply:
MutableCache doesnt have any slots. Is that what you mean?

Reply:
Slots aren't defined on mutable cache